File: AD-HOC/2247/main.py
test = 1
while (n := int(input())):
    print(f'Teste {test}')
    difference = 0
    for _ in range(n):
        j, z = map(int, input().split())
        difference += j - z
        print(difference)
    print()
    test += 1

File: AD-HOC/2247/main.dart
import 'dart:io';

void main() {
  for (int n, test = 1, difference = 0; (n = int.parse(stdin.readLineSync()!)) > 0; test++, difference = 0, print('')) {
    print('Teste $test');
    while (n-- > 0) {
      var [j, z] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
      difference += j - z;
      print(difference);
    }
  }
}
